src/components/templates/TemplateWorkflowsContent.vue

```diff
@@ -1,68 +1,134 @@
 <template>
-  <div
-    class="flex flex-wrap content-around justify-around gap-4 mt-4"
-    data-testid="template-workflows-content"
-  >
-    <div
-      v-for="template in templates"
-      :key="template"
-      :data-testid="`template-workflow-${template}`"
+  <div class="flex h-96">
+    <Listbox
+      v-model="selectedTab"
+      :options="tabs"
+      optionLabel="title"
+      scroll-height="auto"
+      class="listbox"
+      listStyle="max-height:unset"
+    />
+    <Carousel
+      class="carousel justify-center"
+      :value="selectedTab.templates"
+      :numVisible="4"
+      :numScroll="3"
+      :key="selectedTab.moduleName"
     >
-      <Card>
-        <template #header>
-          <div
-            class="relative overflow-hidden rounded-lg cursor-pointer"
-            @click="loadWorkflow(template)"
-          >
-            <img
-              :src="`templates/${template}.jpg`"
-              class="w-64 h-64 rounded-lg object-cover"
-            />
-            <a>
+      <template #item="slotProps">
+        <Card>
+          <template #header>
+            <div class="flex items-center justify-center">
               <div
-                class="absolute top-0 left-0 w-64 h-64 overflow-hidden opacity-0 transition duration-300 ease-in-out hover:opacity-100 bg-opacity-50 bg-black flex items-center justify-center"
+                class="relative overflow-hidden rounded-lg cursor-pointer w-64 h-64"
```

Comment:
The template workflow item here should probably be extracted as a separated Vue component.